### PR TITLE
feat: coin98Wallet support

### PR DESCRIPTION
**Pull request by ghost**
Hey Rainbowkit team,

Opening this PR to support [Coin98 Wallet](https://coin98.com/) wallet.
Coin98 Wallet is the #1 non-custodial, multi-chain wallet, and DeFi gateway, designed to seamlessly connect users to the crypto world in a safe and secure manner. 

Please let me know if there's anything I can do to support this PR and get it merged.

Thanks

### CODE REVIEW

**Comment by DanielSinclair** [comment]
@khiemsoft Still seeing some lint issues

**Comment by khiemsoft** [comment]
> @khiemsoft Still seeing some lint issues

Lint fixed, thanks for your suggestion.

File: site/data/docs/custom-wallet-list.mdx [hunk 1]
```diff
@@ -378,6 +378,17 @@ zerionWallet(options: {
 });
 ```
 
+#### Coin98 Wallet
```
Comment (DanielSinclair) [inline site/data/docs/custom-wallet-list.mdx:381]: Is the `Coin98 Wallet` name still valid, or should this be changed?

File: packages/rainbowkit/src/wallets/walletConnectors/coin98Wallet/coin98Wallet.svg
Comment (DanielSinclair) [inline packages/rainbowkit/src/wallets/walletConnectors/coin98Wallet/coin98Wallet.svg:1]: Please use unrounded corners, resize to 28x28 and run through svgo for minimization

**Review by DanielSinclair — CHANGES_REQUESTED** [review]
Please also add the wallet connector to the `packages/example` dApp for testing

**Comment by DanielSinclair** [comment]
Please also enable changes from maintainers

File: site/data/docs/custom-wallet-list.mdx (hunk 1 above)
Comment (khiemsoft) [reply site/data/docs/custom-wallet-list.mdx:381]: Yes, this name is valid. There is no need to change.

**Comment by khiemsoft** [comment]
> Please also add the wallet connector to the `packages/example` dApp for testing

It's already been submitted in the latest commit. Thanks for your review.

**Comment by khiemsoft** [comment]
> Please also enable edits from maintainers

it's done. Could you please check it?

**Comment by DanielSinclair** [comment]
When testing the wallet, I wasn't able to confirm a connection and got stuck at this screen: 
Screenshot 2023-07-28 at 12 54 58 AM

**Comment by DanielSinclair** [comment]
Does Coin98 support WalletConnect? Currently the `projectId` is accepted, but the WalletConnectConnector is not implemented and `mobile` and `qrCode` params are not returned in the connector creator.

**Comment by khiemsoft** [comment]
> When testing the wallet on Chrome, I wasn't able to confirm a connection and got stuck at this screen: Screenshot 2023-07-28 at 12 54 58 AM

Maybe you click outside the modal and it disappears like this video. Could you please test this by opening the modal state only, not full screen like this?
https://github.com/rainbow-me/rainbowkit/assets/87562184/262b8fb8-a926-4ca2-8067-92cde9e9bea8

**Comment by khiemsoft** [comment]
> Does Coin98 support WalletConnect? Currently the `projectId` is accepted, but the WalletConnectConnector is not implemented and `mobile` and `qrCode` params are not returned in the connector creator.

It's already been submitted in the latest commit. Thanks for your review.

**Comment by DanielSinclair** [comment]
@khiemsoft This is the flow I'm testing. Please QA against this dApp deployment: https://rainbowkit-example-o00ljxf68-rainbowdotme.vercel.app/
Screenshot 2023-08-01 at 3 30 48 PM

**Comment by khiemsoft** [comment]
> @khiemsoft This is the flow I'm testing. Please QA against this dApp deployment: https://rainbowkit-example-o00ljxf68-rainbowdotme.vercel.app/ Screenshot 2023-08-01 at 3 30 48 PM

Hi, this is our re-test video. Could you please record a video or something like that so I can re-test this bug? Thanks for your review. @DanielSinclair 

https://github.com/rainbow-me/rainbowkit/assets/87562184/15cdeaaf-3049-4ff1-9020-06a8e5c88005

**Comment by DanielSinclair** [comment]
@khiemsoft Here is a video displaying the issue. I'll merge this PR, but there is an outstanding issue with the wallet that prevents connections and signatures.

https://github.com/rainbow-me/rainbowkit/assets/4412473/7810aa5c-f5c9-4d60-9247-500844d9df7a

**Comment by DanielSinclair** [comment]
@khiemsoft There also appears to be an issue with WalletConnect support. Please test against the example dApp deployment and reference the console errors returned by WalletConnect.

**Review by DanielSinclair — APPROVED** [review]

**Pull request merged**